packages/news-api/src/server.ts

```diff
@@ -19,15 +19,22 @@ app.get('/', (_req, res) => {
 
 app.get('/posts', (req, res) => {
     const { limit } = req.query;
-    fetcher(limit, (statusCode, data: string | null, errorMsg) => {
+    if (typeof limit !== 'string') {
+        return res
+            .status(400)
+            .end(JSON.stringify({ status: 'error', errorMsg: 'Invalid limit param' }));
+    }
+
+    const limitCount = parseInt(limit, 10);
```

Comment:
I guess this can also throw a 400 error if `limitCount` is `NaN`.

Reply:
I didn't want to mess with 2 ugly returns 😁 fixed